google/cloud/apigee/registry/applications/v1/diff_analytics.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package google.cloud.apigee.registry.applications.v1;
+// [START go_declaration]
+option go_package = "github.com/apigee/registry/rpc;rpc";
+// [END go_declaration]
+
+/* This proto contains the diff of a spec and its revision. Changes that are
+additions fall in the added field. Changes that are Deletions fall in the
+deleted field, and changes that modify values fall in the value_modify map
+field. */
+message Diff {
+  // Change will be displayed in format foo.bar.x
+  repeated string additions = 1;
+  // Change will be displayed in format foo.bar.x
+  repeated string deletions = 2;
+
+  message ValueChange {
+    string From = 1;
```

Comment:
Is there a reason for these to be capitalized? Other fields in our protos are generally lowercase.